views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,2 @@
+
+<div>{{this.name}} <img src="{{this.image_url}}" alt="img"> {{this.description}} {{this.tagline}}</div>
```

Comment:
As best practices, write your div content in separate lines for readability. More important, never write your html without html tags (see description and tagline)